lib/oli/interop/export.ex

```diff
@@ -306,7 +322,8 @@ defmodule Oli.Interop.Export do
     %{
       title: project.title,
       description: project.description,
-      type: "Manifest"
+      type: "Manifest",
+      required_student_survey: project.required_survey_resource_id
```

Comment:
This needs to convert the `required_survey_resource_id` to a string.  All ids in export archives must be strings to accomodate that in the OLI legacy system these strong ids were alphanumeric in nature.